core/src/main/scala/kafka/admin/TopicCommand.scala

```diff
@@ -172,14 +172,19 @@ object TopicCommand extends Logging {
   case class AdminClientTopicService private (adminClient: JAdminClient) extends TopicService {
 
     override def createTopic(topic: CommandTopicPartition): Unit = {
-      if (topic.replicationFactor > Short.MaxValue)
-        throw new IllegalArgumentException(s"The replication factor's maximum value must be smaller or equal to ${Short.MaxValue}")
+      if (topic.replicationFactor.exists(replicas => replicas > Short.MaxValue || replicas < 1))
```

Comment:
nit: the name `replicas` threw me off. Maybe `replicationFactor` or even `rf` would be clearer?